app/packs/src/components/sampleTaskInbox/SampleTaskCard.js

```diff
@@ -155,35 +154,39 @@ const SampleTaskCard = ({ sampleTask }) => {
         <p>Deletion of a Scan Task cannot be undone. Please check carefully</p>
         <p>The task is missing the following to be completed:</p>
         <ul>
-          {sampleTaskStillOpenReasons().map((reason) => (<li>{reason}</li>))}
+          {
+            sampleTaskStillOpenReasons().map((reason, index) => (
+              <li key={`missingStepsForSampleTask_${sampleTask.id}_${index}`}>
+                {reason}
+              </li>
+            ))
+          }
         </ul>
       </div>
     );
   }
 
   return (
-    <Panel variant="info">
-      <Panel.Heading>
+    <Card className="w-100">
+      <Card.Header>
         {panelHeading()}
         {deleteButton()}
-      </Panel.Heading>
-      <Panel.Body>
-        <div className="row">
-          <div className="col-sm-6">
-            {contentForSample(sampleTask, dropRef)}
-          </div>
-          <div className="col-sm-6">
-            {contentForSampleTask(sampleTask)}
-          </div>
+      </Card.Header>
+      <Card.Body className="d-flex">
+        <div className="w-50">
```

Comment:
You can use the Col component directly in a d-flex container.
If you need extra DOM-elements for layout purpose I would suggest using the dedicated components (Container, Row, Col).
Apply the corresponding utility classes, when they reduce nesting depth.
Col components (and the simple .col class) automatically take up equal parts of their parent container – no "w-50" class needed.
https://react-bootstrap.netlify.app/docs/layout/grid

Reply:
The result is pretty ugly markup-wise, but okay 🙈